dcpy/utils/versions.py

```diff
@@ -120,11 +120,47 @@ def generate() -> FirstOfMonth:
                 raise Exception("Version parsing failed")
 
 
+@dataclass(order=True)
```

Comment:
Ooh. This is going to collide with the current "first of month". All our other "Today" ones are formatted "{year}{month}{day}"

Maybe worth biting the bullet and making "monthly" rather than "first of month" for ztl, and format it "{year}-{month}". Or standardizing it as a "major" build every month. Food for thought. Though it'll need to be addressed here in some way so we don't have ambiguous versions.

Or we could leave KPDB for now, which I thought we had said was the plan yesterday?

Reply:
yea I also hit some other fun stuff [here](https://github.com/NYCPlanning/data-engineering/blob/5cfe42977415bb6fdc2591fead1aa6e37cc16468/dcpy/test/utils/test_versions.py#L38) with date math, bumping date-based versions could probably use a refactor to use the `datetime` library or something